Apps/phvictorops/readme.html

```diff
@@ -10,13 +11,13 @@
 <head></head>
 
 <body>
-    To get started with the VictorOps API to integrate with Phantom, log into the VictorOps web portal and go to Settings then
+    To get started with the VictorOps API to integrate with Phantom, log into the VictorOps web portal and go to Integrations then
     API. From here you can retrieve your API ID and have the opportunity to create your API Keys. Note: Only VictorOps admin
     users can create API Keys. The documentation for the VictorOps API allows you to try out the different calls in real
-    time. To get started select Read the API Documentation link on the API settings page.<br /><br />Phantom integrates with
-    the VictorOps REST Endpoint to trigger, update, or resolve incidents in VictorOps. To enable this integration
-    in VictorOps, click on Settings >> Alert Behavior >> Integrations >> REST – Generic. If the REST endpoint integration
-    has not been enabled, click the blue Enable button to generate your endpoint destination URL.
+    time. To get started Read the <a href="https://portal.victorops.com/api-docs/#/" target="_blank">API Documentation</a>.
```

Comment:
```suggestion
    time. To get started, read the <a href="https://portal.victorops.com/api-docs/#/" target="_blank">API Documentation</a>.
```

Reply:
HI @tonyc-phantom 
Thanks for the suggestions. We have done the requested change.